### ISSUE TITLE
Dos go-carbon support tags?

### ISSUE THREAD

**Issue by viq** [go-graphite/go-carbon#211]
Is [this recent feature](https://graphite.readthedocs.io/en/latest/tags.html) supported by `go-carbon`?

**Comment by lomik** [comment]
Tag support is not planned in the near future.

**Comment by lomik** [comment]
PR are welcome ;)

**Comment by lomik** [comment]
It was not planned... But added in 0.12.0-rc1

**Comment by deniszh** [comment]
👍

**Comment by lomik** [comment]
Released in 0.12.0

**Comment by Quentin-M** [comment]
@lomik What does this include exactly? While I can obviously write tagged metrics, graphite-web never seem to be able to query them back through carbonserver. I don't seem to find any reference to tags in the code path used by /render either. This is a real bummer for us not to be able to use go-carbon in production for the query side, and be forced to use the terribly sluggish graphite-web.

```
$ echo "local.random.diceroll;sv=backend10 1 `date +%s`" | nc localhost 2002

[2018-09-28T23:22:16.419-0700] DEBUG [whisper:new] created {"path": "data/_tagged/67e/2d4/local_DOT_random_DOT_diceroll;sv=backend10.wsp", "retention": "10s:14d,1m:90d,30m:1y,1h:5y", "schema": "default", "aggregation": "default_average", "xFilesFactor": 0.5, "method": "average"}
```

```
$ curl -vv -F "format=json" -F "local=1" -F "noCache=1" -F "from=1536704479" -F "until=1536715279" -F "now=1538197298" -F "target=local.random.diceroll;sv=backend10" "localhost:8080/render/"

[2018-09-28T23:22:28.072-0700] ERROR [access] fetch failed {"handler": "render", "url": "/render/", "peer": "[::1]:60439", "format": "json", "targets": ["local.random.diceroll;sv=backend10"], "runtime_seconds": 0.000586684, "reason": "no metrics found", "http_code": 404}

lstat64("...data/local/random/diceroll;sv=backend10.wsp\0", 0xC4201BED38, 0x0)		 = -1 Err#2
lstat64(...data/local/random/diceroll;sv=backend10\0", 0xC4201BEE08, 0x0)		 = -1 Err#2
```
